MetaCell/cloud-harness
# Feature/192 update server applications

**Manual testing:**
* Run harness-deployment and check if images build with success
* Install helm chart in minikube and check if applications start and show no error in logs

Closes #194
